src/components/Slider/Slider.tsx

```diff
@@ -135,15 +147,30 @@ export const Slider = ({
         id={sliderId}
         max={max}
         min={min}
+        ref={ref}
         step={step}
         style={
           // calculate value as a ratio to color the track
-          { '--ratio': (value - min) / (max - min) } as CSSProperties
+          { '--ratio': ratio } as CSSProperties
         }
         type="range"
         value={value}
         {...other}
       />
+      {tooltip && (
+        <Tooltip
+          align="top"
+          appendTo="parent"
+          hideOnClick={false}
+          offset={({ reference }) => {
+            // offsets the tooltip relative to the position of the thumb
+            // the magic number 24 is the slider thumb size (see relevant css module)
+            return [(ratio - 0.5) * (reference.width - 24), 0];
+          }}
+          reference={ref}
+          text={tooltip}
+        />
+      )}
```

Comment:
Ideally we wouldn't have Tooltip functionality baked into the Slider component like this, but the native `<input type="range">` makes the thumb a shadow dom element and unselectable for the Tooltip.

I did explore having an invisible, absolutely positioned div on top of the thumb, but ran into a couple issues:
1. The div on top would need to pass mouse functionality to the thumb below, but with `pointer-events: none` removes the hover on it to spawn the tooltip
2. If we forward a ref to the tooltip, I feel like it should be on the input element and not the invisible div. That can be somewhat resolved with `useImperativeHandle` to have two refs but that means moving two refs around and also increases complexity too much for this, and seems like a lot of boilerplate to be writing for the consumer

Reply:
Some thoughts to mitigate would be to no longer use the semantic element and handle the various accessibility requirements ourselves and hence have a referencable thumb... but that's outside current scope